scenes/enumerated/problem_11018/problem_11018.py

```diff
@@ -0,0 +1,177 @@
+from manim import Write, FadeOut, FadeIn,AnimationGroup, Tex, VGroup, TransformFromCopy,ReplacementTransform,Transform
+from manim import Arrow, CurvedArrow, SurroundingRectangle
+from manim import YELLOW, ORANGE, WHITE, MathTex
+from manim import Scene, LEFT, UP, DOWN, RIGHT
+from qarakusiscene import TaskNumberBox
+from .text import *
+
+FONT_SIZE = 70
+RUN_TIME_SPEED = 2
+
+class Problem11018(Scene):
+    def construct(self):
+        scale_factor=1.2
+        product_rows=VGroup(
+            MathTex('*\;2\;* ',font_size=FONT_SIZE).shift(UP*2),
+            MathTex(r' \frac{*\;7}{*\;*\;*\;}', font_size=FONT_SIZE).shift(UP*1/1.2),
+            MathTex(r' \frac{*\;*\;*\;*}{*\;*\;*\;*\;8}', font_size=FONT_SIZE).shift(DOWN),
+        ).shift(LEFT*0.5)
+
+        #arrange all digits
+        product_rows[1][0][0:2].align_to(product_rows[0][0],RIGHT)
+
+        for i in range(len(product_rows[0][0])):
+            product_rows[0][0][i].align_to(product_rows[1][0][-3+i], RIGHT)
+
+        product_rows[1][0][0].align_to(product_rows[0][0][1],RIGHT)
+        product_rows[1][0][1].align_to(product_rows[0][0][2],RIGHT)
+
+        product_rows[2][0].align_to(product_rows[1][0],RIGHT)
+
+        for i in range(-1,-4,-1):
+            product_rows[2][0][i].align_to(product_rows[1][0][i], RIGHT)
+
+        product_rows[2][0][3].align_to(product_rows[1][0][-2], RIGHT)
+        product_rows[2][0][2].align_to(product_rows[1][0][-3], RIGHT)
+        product_rows[2][0][0].align_to(product_rows[2],LEFT)
+        product_rows[2][0][1].align_to(product_rows[2],LEFT).shift(RIGHT*0.65)
+
+
+        product_rows[2][0][-5].align_to(product_rows[2], LEFT)
+        product_rows[2][0][-4].align_to(product_rows[2][0][1],RIGHT)
+
+
+
+        product_expressions=VGroup(MathTex(r'\times',font_size=FONT_SIZE).next_to(product_rows[0],LEFT).shift(DOWN*0.5+LEFT*0.1),
```

Comment:
Can you add a space before and after assignment (`=`) it will be easier to see on github and most of IDEs/editors